README.md

```
@@ -187,12 +189,14 @@ export MAGICK_HOME=/opt/homebrew
 - Configure Postgres to ensure that password authentication is enabled for the gooey user
   - open the pg_hba.conf file in a text editor. On Linux, by default, it is usually located either at `/etc/postgresql/<version>/main/` or `/var/lib/pgsql/<version>/data/`
   - add/edit the file so that there are lines at the bottom that looks like this:
+
   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)